src/collections/symbol-collection.ts

```diff
@@ -57,7 +57,7 @@ export class SymbolCollection {
     }
 
     const symbolNames = new Set(Object.values(this.symbolNames));
-    symbolName = toPascalCase(id.replace(/\./g, '_'));
+    symbolName = toPascalCase(id.replace(/\./g, '_').replace(/-/g, '_'));
```

Comment:
Thanks for the contribution!

We could do this in one pass:

```suggestion
    symbolName = toPascalCase(id.replace(/[.-]/g, '_');
```

But perhaps we should really replace all characters that aren't allowed in JavaScript identifiers:

```suggestion
    symbolName = toPascalCase(id.replace(/[^a-zA-Z$_0-9]/g, '_');
```

And then add a prefix underscore if the identifier starts with `[0-9]`... probably needs a few tests so the logic is correct.